modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -596,10 +611,18 @@ public class IgniteImpl implements Ignite {
                 logicalTopologyService,
                 raftMgr,
                 topologyAwareRaftGroupServiceFactory,
-                clockService
-        );
+                clockService,
+                tablePartId -> {
+                    CatalogTableDescriptor tbl = catalogManager.table(tablePartId.tableId(), catalogManager.latestCatalogVersion());
 
-        ReplicationConfiguration replicationConfig = clusterConfigRegistry.getConfiguration(ReplicationConfiguration.KEY);
+                    int zoneId = tbl == null ? 2 : tbl.zoneId();
```

Comment:
Why `2`? Should be there some named constant at least?

Reply:
It is a really complicated question for me. 
Here, I try to return a default group id, because the default group always exists and the table probably belongs to default. Finally, nobody promised that the placement driver might not make a mistake.
But currently, we do not know an ID for the default replication group. I think it has to be a constant, otherwise the consequences of this might be freezing.

Reply:
Then, may be to introduce `DistributedZone.DEFAULT_ZONE_ID`?

Reply:
Right now, we do not have a specific constant. We should create a specific ticket to define it.
Any way, it is a temporary solution that would be fixed before merging to main.